web/state/Roles.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+const ZERO = 0;
+const ERR_NO_ROLE_NAME = 1;
+const ERR_SPACE_IN_ROLE_LABEL = 2;
+const ERR_ROLE_ALREADY_EXISTS = 3;
+
+class Roles {
+  constructor (bdk, botName) {
+    this.roomId = bdk.getRoomId();
+    this.botName = botName;
+    this.bdk = bdk;
+    this.ROLES_BOTDATA_NAME = 'assignedParticipants';
```

Comment:
https://github.com/salesforce/refocus-bot-participants/blob/refactor-and-assignedParticipants/service/constants.js#L1
The constant for 'assignedParticipants' is not in use, should we remove it?

Reply:
Yep, going to set this as a constant at the top of the file as well as mentioned in the discussion